### PR TITLE
CHAIN-250: update icons to include chain name

### PR DESCRIPTION
**Pull request by bholzman**
Also added the chain name in text under the symbol name in the symbol selector on the swap screen (but not yet on the Dashboard)

Screenshot 2024-05-29 at 11 59 54 PM
Screenshot 2024-05-29 at 11 59 58 PM
Screenshot 2024-05-30 at 12 00 08 AM

### CODE REVIEW

File: scripts/envs/demo.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ environment:
     NATIVE_CURRENCY_NAME: "Demo Botanix BTC"
   mocker:
     API_URL: http://demo-api.chainring.co:9000
-    EVM_CHAINS: anvil,anvil2
+    EVM_CHAINS: Bitlayer,Botanix
     EVM_NETWORK_URL_anvil: http://demo-anvil.chainring.co:8545
     EVM_NETWORK_URL_anvil2: http://demo-anvil2.chainring.co:8545
```
Comment (imykolenko) [inline scripts/envs/demo.yml:33]: anvil/anvil2 also needs to be replaced

File: web-ui/src/components/common/SubmitButton.tsx [hunk 2]
```diff
@@ -4,12 +4,14 @@ export default function SubmitButton({
...
   caption,
-  error
+  error,
+  success
 }: {
   disabled: boolean
   onClick: () => void
   caption: () => string
   error: string | null | undefined
+  success: boolean
```
Comment (imykolenko) [inline web-ui/src/components/common/SubmitButton.tsx:14]: It is not immediately clear what `success={false}` passed to a button component means. What if it is renamed to appearance with values `'green' | 'blue'`?

**Review by imykolenko — APPROVED** [review]

File: web-ui/src/components/common/SubmitButton.tsx (hunk 2 above)
Comment (antonyshchenko) [reply web-ui/src/components/common/SubmitButton.tsx:14]: Or perhaps we could use the presence or absence of `error` to defer the `success` state.

Comment (bholzman) [reply web-ui/src/components/common/SubmitButton.tsx:14]: Nice, I like that, will do!

File: scripts/envs/demo.yml (hunk 1 above)
Comment (bholzman) [reply scripts/envs/demo.yml:33]: Oops, thanks!

**Pull request merged**